packages/react-core/src/components/ContextSelector/ContextSelector.tsx

```diff
@@ -92,12 +95,15 @@ export class ContextSelector extends React.Component<ContextSelectorProps> {
       menuAppendTo,
       ouiaId,
       ouiaSafe,
+      beforeSearch,
       ...props
     } = this.props;
+
     const menuContainer = (
       <div className={css(styles.contextSelectorMenu)}>
         {isOpen && (
           <FocusTrap focusTrapOptions={{ clickOutsideDeactivates: true }}>
+            <ContextSelectorMenuList isOpen={isOpen}>{beforeSearch}</ContextSelectorMenuList>
```

Comment:
You might want to check that you have something you want to render here before you include the `ContextSelectorMenuList`. if the prop is undefined or null, then you probably don't want to include the element.

Reply:
Yeah you're right, thanks! added a `beforeSearch && <ContextSelectorMenuList...`